feature/better-oss (#1)
mntorg/mnt-server: README.md

# MENT Backend Service

> A robust FastAPI-based backend service powering MENT, a social platform that combines task-based interactions with social networking features. The service handles user verification, real-time chat, task management, and media processing through a microservices architecture.

Reviewer: A robust FastAPI-based backend service powering MENT, a location based social network. The service handles user verification, real-time chat, task management, and media processing through a microservices architecture.
```suggestion
A robust FastAPI-based backend service powering MENT, a location based social network. The service handles users feed, real-time chat, and media processing through a microservices architecture.
```

Reviewer: task-based interactions ძველი ამ კოდის feature ია რომელიც აღარ გამოიყენება დამავიწყდა ამოღება ამ კოდის.
